.github/workflows/ci.yml

```diff
@@ -109,7 +109,7 @@ jobs:
 
       - name: Fetch sources for sibling projects
         run: |
-          git clone --depth=50 --branch=master https://github.com/SasView/sasdata.git ../sasdata
+          git clone --depth=50 --branch=SasData-Counterpart-to-Sasview-SlicerExtension_1344 https://github.com/SasView/sasdata.git ../sasdata
```

Comment:
For record-keeping purposes: This line needs to be reset to `master` once the associated sasdata PR is merged.

Reply:
The two associated PR should be merged together. But agreed this file should NOT be part of the merge. It will probably get cleaned up in the rebasing